tutorials/Tutorial For QA Registration.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QA REGISTRATION\n",
+    "Give a visual result for regisration part to help user check results of each step. The graph includes three dimensions(sagital slice, coronal slice and axial slice) and uses red and green to indicate the rigistered output and reference image."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup Steps\n",
+    "qa_reg is used to do quality assurance for registration part where some of its functions are included in qa_utils and gen_utils."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/xueminzhu/anaconda3/lib/python3.7/site-packages/sklearn/externals/joblib/__init__.py:15: DeprecationWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.\n",
+      "  warnings.warn(msg, category=DeprecationWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import nibabel as nb\n",
+    "from ndmg.utils.gen_utils import get_braindata\n",
+    "from ndmg.utils.qa_utils import get_min_max, opaque_colorscale, pad_im\n",
+    "from argparse import ArgumentParser\n",
+    "from scipy import ndimage\n",
+    "from matplotlib.colors import LinearSegmentedColormap\n",
+    "from nilearn.plotting.edge_detect import _edge_map as edge_map\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generating the qa overlay figures"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_overlays(atlas, b0, cmaps=None, minthr=2, maxthr=95, edge=False):\n",
+    "    \"\"\"\n",
+    "    A function to plot the overlay figures of registered and reference brain slices.\n",
+    "    \n",
+    "    Parameter\n",
+    "    ---------\n",
+    "    atlas: str, nifti image, numpy.ndarray\n",
+    "        an object to open the data for a registered brain. Can be a string (path to a brain file),\n",
+    "        nibabel.nifti1.nifti1image, or a numpy.ndarray.\n",
+    "\n",
+    "    b0: str, nifti image, numpy.ndarray\n",
+    "        an object to open the data for a reference brain. Can be a string (path to a brain file),\n",
+    "        nibabel.nifti1.nifti1image, or a numpy.ndarray.\n",
+    "\n",
+    "    cmap: Colormap objects based on lookup tables using linear segments.\n",
+    "    minthr: int\n",
+    "    maxthr: int\n",
+    "    edge: bool\n",
```

Comment:
Add description for these inputs

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/a48b23981fbece4dcac6471538b1f2b786275ef1